src/test/java/uk/gov/pay/ledger/report/service/ReportServiceTest.java

```diff
@@ -36,32 +38,33 @@
     private ReportDao mockReportDao;
 
     @Captor
-    private ArgumentCaptor<TransactionSummaryParams> transactionSummaryParamsArgumentCaptor;
+    private ArgumentCaptor<TransactionStatisticQuery> transactionStatisticQueryArgumentCaptor;
 
     @InjectMocks
     private ReportService reportService;
 
     @Test
     public void shouldReturnPaymentCountsByState_whenGatewayAccountIdProvided() {
```

Comment:
should probably add in service tests for whether or not we're including moto statistics too